Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -295,18 +295,47 @@ Describe "New-GcsObject" {
         Remove-GcsObject $emptyObj
     }
 
+    It "should work with fixed-type metadata" {
```

Comment:
Would it be better to write this as a unit test than an integration test?

Reply:
I needed to test that the object returned had the metadata in the correct place.